src/test/java/net/sf/jabref/exporter/layout/format/HTMLCharsTest.java

```diff
@@ -21,14 +21,14 @@ public void testBasicFormat() {
 
         Assert.assertEquals("h&aacute;llo", layout.format("h\\'allo"));
 
-        Assert.assertEquals("&#305; &#305;", layout.format("\\i \\i"));
-        Assert.assertEquals("&#305;", layout.format("\\i"));
-        Assert.assertEquals("&#305;", layout.format("\\{i}"));
-        Assert.assertEquals("&#305;&#305;", layout.format("\\i\\i"));
+        Assert.assertEquals("&imath; &imath;", layout.format("\\i \\i"));
```

Comment:
Why does a HTMLChars() formatter format latex to unicode and not to HTML?
Is there something wrong with the implementation or is the naming just suboptimal?